platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/internal/artifacts/configurations/DefaultConfigurationExtendsFromIntegrationTest.groovy

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.internal.artifacts.configurations
+
+import org.gradle.api.internal.artifacts.configurations.DefaultConfiguration
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.util.GradleVersion
+
+/**
+ * Integration tests for {@code extendsFrom} method in {@link DefaultConfiguration}.
+ */
+class DefaultConfigurationExtendsFromIntegrationTest extends AbstractIntegrationSpec {
```

Comment:
We already have a `ExtendingConfigurationsIntegrationTest`

Reply:
Well...that's just poor luck that it didn't come up when I searched for "Extends" then.  I'll consolidate these.